Library/Formula/csfml.rb

```diff
@@ -16,8 +16,7 @@ class Csfml < Formula
   depends_on "sfml"
 
   def install
-    cp_r "#{Formula["sfml"].share}/SFML/cmake/Modules/", "cmake"
-    system "cmake", ".", *std_cmake_args
+    system "cmake", ".", *std_cmake_args, "-DCMAKE_MODULE_PATH=#{Formula["sfml"].share}/SFML/cmake/Modules/"
```

Comment:
`*std_cmake_args` needs to be the last argument on Ruby 1.8 (which is used by Mountain Lion and below)